task/orka-modular/0.1/README.md

```diff
@@ -0,0 +1,255 @@
+# Orka Tasks
+
+These `Tasks` are for utilizing multiple macOS build agents, in parallel or in series, in your [Orka environment](https://orkadocs.macstadium.com).
+
+## `orka-modular`
+
+This `Task` combines the following three modular sub-tasks: `orka-init`, `orka-deploy`, and `orka-teardown`.
```

Comment:
`nit`: Since this isn't really a single `Task`, but 3 separate Tasks, suggest maybe slightly re-wording this to reflect that.